quiche/src/lib.rs

```diff
@@ -1446,6 +1447,12 @@ pub struct Connection {
 
     /// Structure used when coping with abandoned paths in multipath.
     dcid_seq_to_abandon: VecDeque<u64>,
+
+    /// Pending Path Events that needs to be passed on paths when they call
+    pending_per_path_events: HashMap<(SocketAddr, SocketAddr), VecDeque<path::PathEvent>>,
```

Comment:
Is a sockaddr tuple the best way to identify a path? I think it is effectively forbidden in the current mp spec to have several paths sharing a 4-tuple or to have path migration but just to check